R/loadReg2.R

```diff
@@ -393,7 +393,7 @@ predictSolute.loadReg2 <- function(
 #' @family simulateSolute
 simulateSolute.loadReg2 <- function(load.model, flux.or.conc=c("flux","conc"), newdata, 
                                     method=c("parametric", "non-parametric"), from.interval=c("confidence", "prediction"), rho, ...) {
-  
+  s.hat=NULL
```

Comment:
as with simulateSolute.lm, i think this is real. please be careful to understand the cause of the error as fully as possible before making corrections - you could have noticed the bug, too. it's a similar fix to that for simulateSolute, but i'll need to think about it some more to get it right. in the meantime, please delete the `s.hat=NULL` line and insert this line just before `noise <- noise * s.hat/sd(noise)`:
```r
error("need to calculate s.hat")
```
then you can comment out the `noise <- noise * s.hat/sd(noise)` line to get rid of the R CMD check error